adminSiteServer/plainRouterHelpers.tsx

```diff
@@ -0,0 +1,105 @@
+import { NextFunction, Request, Response, Router } from "express"
+import * as db from "../db/db.js"
+export function getPlainRouteWithROTransaction<T>(
```

Comment:
I wasn't even aware of the plain/functional router distinction!

Not for the PR of course, but do you think it's worth keeping both around?

Reply:
Yeah I think we could clean that up but it doesn't seem particularly urgent